### PR TITLE
[DOC-7078] Review of intro to querying

### PR DESCRIPTION
**Pull request by urbiz-nr**
- new intro
- new flow
- more examples

See DOC-7078 for more details.

### CODE REVIEW

File: src/content/docs/query-your-data/explore-query-data/get-started/introduction-querying-new-relic-data.mdx [hunk 1]
```diff
@@ -21,46 +21,101 @@ redirects:
...
+* [Query builder in PromQL-style mode](/docs/query-your-data/explore-query-data/chart-builder/use-advanced-promql-mode-specify-data): Write basic queries using a PromQL-style query.
 
 ## Query data via API [#query-apis]
 
-You can use APIs to retrieve and query your data in New Relic. For example, you can run [NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language) (our query language) queries with [NerdGraph](/docs/apis/graphql-api/get-started/introduction-new-relic-nerdgraph) (our GraphQL API).
+For use cases where getting into the New Relic platform is not an option, you can use APIs to retrieve and query your data in New Relic. For example, you can run [NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language) (our query language) queries with [NerdGraph](/docs/apis/graphql-api/get-started/introduction-new-relic-nerdgraph) (our GraphQL API).
```
Comment (x8a) [inline src/content/docs/query-your-data/explore-query-data/get-started/introduction-querying-new-relic-data.mdx:119]: I'd rephrase this to `When getting into the New Relic platform is not an option,`. This way we're more direct.

**Review by x8a — DISMISSED** [review]
Looks amazing!

File: src/content/docs/query-your-data/explore-query-data/get-started/introduction-querying-new-relic-data.mdx (hunk 1 above)
Comment (urbiz-nr) [reply src/content/docs/query-your-data/explore-query-data/get-started/introduction-querying-new-relic-data.mdx:119]: Good call, thanks!

**Review by x8a — APPROVED** [review]
